sc/src/main/java/uk/ac/ebi/atlas/search/JsonBioentityInformationController.java

```diff
@@ -51,15 +43,8 @@ public String getBioentityInformation(@PathVariable String geneId) {
                 bioEntityPropertyDao.fetchPropertyValuesForGeneId(geneId, BioentityPropertyName.SYMBOL).stream()
                         .collect(Collectors.joining("/"));
 
-
         Map<String, Object> model = bioEntityCardModelFactory.modelAttributes(geneId, species, BioEntityCardProperties.bioentityPropertyNames, geneName, propertyValues);
 
         return model.get("bioentityProperties").toString();
```

Comment:
So far we’ll leave it there. It’s sort of ugly but it’ll do. Ideally we can refactor the `bioentityProperties` and explicitly JSON-serialize it instead of having a `toString` method which is semantically confusing. This will become its own thing when we deprecate the current model and both EA and SCEA use this endpoint and the new component.